connector/src/snapshot.rs

```diff
@@ -55,50 +77,22 @@ pub async fn get_snapshot_gma(
         min_context_slot: None,
     };
 
-    info!("requesting snapshot {:?}", ids);
-    let account_snapshot = rpc_client
+    info!("requesting gma snapshot {:?}", ids);
+    let account_snapshot_response = rpc_client
         .get_multiple_accounts(ids.clone(), Some(account_info_config))
```

Comment:
future: gma typically has an upper limit, might need to split `ids` and issue multiple parallel requests

Reply:
the code in _rpc.rs_ does not impose a limit AFAIK